apps/tangle-dapp/src/containers/bridge/BridgeContainer.tsx

```diff
@@ -426,18 +426,19 @@ const BridgeContainer = ({ className }: Props) => {
 
   const assets: AssetConfig[] = useMemo(() => {
     return tokens.map((token) => {
```

Comment:
Utilize AI to refactor this `assets` calculation memo for better readability and easier review.